noir/Earthfile

```diff
@@ -201,6 +201,45 @@ build-acir-tests:
     RUN ./rebuild.sh true
     SAVE ARTIFACT /usr/src/noir-repo/test_programs/acir_artifacts/*
 
+barretenberg-acir-benches-bb:
+  FROM ../build-images/+build
+
+  COPY ../barretenberg/cpp/+preset-clang-assert/bin/bb /usr/src/barretenberg/cpp/build/bin/bb
+  COPY ../barretenberg/+acir-tests/ /usr/src/barretenberg/acir_tests
+  COPY +build-acir-tests/ /usr/src/acir_artifacts
+
+  ENV TEST_SRC=/usr/src/acir_artifacts
+
+  WORKDIR /usr/src/barretenberg/acir_tests
+
+export-barretenberg-acir-benches-bb:
+  ARG EARTHLY_GIT_HASH
+  FROM +barretenberg-acir-benches-bb
+  SAVE IMAGE aztecprotocol/barretenberg-acir-benches:$EARTHLY_GIT_HASH
+
+FOO:
```

Comment:
Not sure if this slipped by or part of some ongoing testing

Reply:
This is some ongoing testing atm. Mostly trying to tease out the cause for different behviour I'm seeing between this vs the e2e tests.